<ui version="4.0" >
 <class>Meswak</class>
 <widget class="QMainWindow" name="Meswak" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Meswak Toothpaste</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="Meswak.qrc" >:/new/prefix1/Meswak.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QFrame" name="glframe" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>144</height>
         </size>
        </property>
        <property name="title" >
         <string>Controls</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>X Rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sliderXRot" >
             <property name="minimum" >
              <number>-180</number>
             </property>
             <property name="maximum" >
              <number>180</number>
             </property>
             <property name="value" >
              <number>-45</number>
             </property>
             <property name="sliderPosition" >
              <number>-45</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>Y Rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sliderYRot" >
             <property name="minimum" >
              <number>-180</number>
             </property>
             <property name="maximum" >
              <number>180</number>
             </property>
             <property name="value" >
              <number>0</number>
             </property>
             <property name="sliderPosition" >
              <number>0</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" >
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Z Rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sliderZRot" >
             <property name="minimum" >
              <number>-180</number>
             </property>
             <property name="maximum" >
              <number>180</number>
             </property>
             <property name="value" >
              <number>-45</number>
             </property>
             <property name="sliderPosition" >
              <number>-45</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0" >
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4" >
             <property name="text" >
              <string>Zoom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sliderZoom" >
             <property name="minimum" >
              <number>1</number>
             </property>
             <property name="maximum" >
              <number>200</number>
             </property>
             <property name="value" >
              <number>50</number>
             </property>
             <property name="sliderPosition" >
              <number>50</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiel" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionTop" />
    <addaction name="actionBottom" />
    <addaction name="actionFront" />
    <addaction name="actionBack" />
    <addaction name="actionLeft" />
    <addaction name="actionRight" />
    <addaction name="separator" />
    <addaction name="actionAxonometric" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuAnimation" >
    <property name="title" >
     <string>Animation</string>
    </property>
    <addaction name="actionAnimStop" />
   </widget>
   <addaction name="menuFiel" />
   <addaction name="menuView" />
   <addaction name="menuAnimation" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTop" >
   <property name="text" >
    <string>Top</string>
   </property>
  </action>
  <action name="actionBottom" >
   <property name="text" >
    <string>Bottom</string>
   </property>
  </action>
  <action name="actionFront" >
   <property name="text" >
    <string>Front</string>
   </property>
  </action>
  <action name="actionBack" >
   <property name="text" >
    <string>Back</string>
   </property>
  </action>
  <action name="actionLeft" >
   <property name="text" >
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight" >
   <property name="text" >
    <string>Right</string>
   </property>
  </action>
  <action name="actionAxonometric" >
   <property name="text" >
    <string>Axonometric</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionAnimStop" >
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Meswak.qrc" />
 </resources>
 <connections/>
</ui>
